Installing-and-Configuring/RecoveryShell.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: default
+title: Heads Recovery Shell
+permalink: /RecoveryShell/
+nav_order: 90
+has_children: false
+has_toc: false
+parent: Installing and configuring
+---
+
+Heads Recovery Shell
+====
+
+The recovery shell is a command line where heads and *nix utilities may be accessed.  This document is a work in progress. 
+
+
+Overview
+---
+
+The recovery shell is running a Linux kernel and shell based on busybox.  The environment comes from the initrd image flashed into the BIOS with the Linux kernel.  i.e.  heads
+
+The shell can be used to troubleshoot heads.  The security dongle paired with heads may be needed to effectively use this environment.
+
+
+Access
+----
+
+The recovery shell may be accessed on boot by repeatedly pressing 'r' during boot or by choosing the appropriate menu in the heads GUI.  These two different methods of access will result in some different settings.
+
+
+Limitations
+----
+
+The recovery shell wipes secrets--normally used for security checks--that were [computed](/Keys/#tpm-pcrs) from the BIOS, kernel modules loaded, etc. This will limit the features available in this environment.
```

Comment:
This will limits sealing/unsealing functions (Disk Unlock Key creation, TOTP/HOTP sealing) from the recovery shell environment. To seal/unseal secrets, the same measurements needs to be calculated, which would be different depending of the kernel modules loaded and if going in/out of the recovery shell, which invalidates per design the TPM measurements to not release secrets.

To seal/unseal secrets, use the GUI environment.